packages/neuron-ui/src/components/NervosDAODetail/CellsCard/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { FC, useEffect, useMemo, useState } from 'react'
+import Tabs, { VariantProps } from 'widgets/Tabs'
+import { clsx, isSuccessResponse, localNumberFormatter, shannonToCKBFormatter } from 'utils'
+import { useTranslation } from 'react-i18next'
+import { scriptToAddress } from '@nervosnetwork/ckb-sdk-utils'
+import { onEnter } from 'utils/inputDevice'
+import { EyesClose, EyesOpen } from 'widgets/Icons/icon'
+import { getAllNetworks, getCurrentNetworkID } from 'services/remote'
+import { MAINNET_TAG } from 'utils/const'
+import ScriptTag from 'components/ScriptTag'
+import LockInfoDialog from 'components/LockInfoDialog'
+import styles from './cellsCard.module.scss'
+
+const TabsVariantWithCellsCard: FC<
+  VariantProps<{
+    title: string
+    cells: (State.DetailedInput | State.DetailedOutput)[]
+  }>
+> = ({ tabs, selectedTab, onTabChange }) => {
+  const [t] = useTranslation()
+  const [isPrivacyMode, setIsPrivacyMode] = useState(false)
+  const [isMainnet, setIsMainnet] = useState(false)
+  const [showingLockInfo, setShowingLockInfo] = useState<CKBComponents.Script | null>(null)
+
+  useEffect(() => {
+    Promise.all([getAllNetworks(), getCurrentNetworkID()])
+      .then(([networksRes, idRes]) => {
+        if (isSuccessResponse(networksRes) && isSuccessResponse(idRes)) {
+          const network = networksRes.result.find((n: any) => n.id === idRes.result)
+          if (!network) {
+            throw new Error('Cannot find current network in the network list')
+          }
+
+          setIsMainnet(network.chain === MAINNET_TAG)
+        }
+      })
+      .catch(err => console.warn(err))
+  }, [])
+
+  return (
+    <div className={styles.cells}>
+      <LockInfoDialog lockInfo={showingLockInfo} isMainnet={isMainnet} onDismiss={() => setShowingLockInfo(null)} />
+      <div className={styles.cellTypes} role="tablist">
+        {tabs.map(tab => (
+          <div
+            key={tab.id}
+            className={clsx(styles.cellType, { [styles.active]: selectedTab.id === tab.id })}
+            role="tab"
+            tabIndex={0}
+            onKeyDown={onEnter(() => onTabChange(tab.id))}
+            onClick={() => onTabChange(tab.id)}
+          >
+            {tab.title}
+          </div>
+        ))}
+      </div>
```

Comment:
What is different from this with the widget `Tabs`? Can we use the widget `Tabs`?

Reply:
This is the purpose of the `Variant` property in my design of `Tabs`, to make `Tabs` more like a headless component, which can avoid multiple different custom structures being written into the implementation of `Tabs` itself and making it difficult to maintain.

Reply:
So why not use your `Tabs`?

Reply:
updated